src/core/variable.jl

```diff
@@ -204,3 +204,28 @@ function variable_tp_oltc_tap(pm::PMs.GenericPowerModel; nw::Int=pm.cnw, bounded
         end
     end
 end
+
+
+"Power drawn by the load at the bus to which it is connected"
+function variable_load_flow(pm::PMs.GenericPowerModel; nw::Int=pm.cnw, cnd::Int=pm.ccnd)
+    variable_active_load_flow(pm, nw, cnd)
+    variable_reactive_load_flow(pm, nw, cnd)
+end
+
+
+"Active power drawn by the load at the bus to which it is connected"
+function variable_active_load_flow(pm::PMs.GenericPowerModel, nw::Int, cnd::Int)
```

Comment:
Should these be bounded from 0 to given value?  E.g. see the implementation of dispatchable loads in PowerModels.

Reply:
[This](https://github.com/lanl-ansi/PowerModels.jl/blob/f5289fcee04b4eba0f2a0221c729fb3efb5f74e5/src/core/variable.jl#L286) is the relevant line in the dispatchable branch I think. Why is the upperbound fixed to 1.0?

I can infer bounds via the nominal power drawn, and the voltage magnitude bounds for the bus to which the load is connected. So for example for the constant current load
P = (P0/V0).|V|, so
if P>=0
    P \in [(P0/V0).vmin, (P0/V0).vmax]
else
    P \in [(P0/V0).vmax, (P0/V0).vmin]